cybershell/adaptive/train.py

```
@@ -0,0 +1,49 @@
+    labels: Dict[str, int]
+def load_nvd_json_folder(folder: str) -> List[Sample]:
+    samples: List[Sample] = []
+    for path in sorted(glob.glob(os.path.join(folder, "nvdcve-1.1-*.json"))):
+        with open(path, 'r', encoding='utf-8') as f:
+            data = json.load(f)
+        for it in data.get('CVE_Items', []):
+            descs = it.get('cve',{}).get('description',{}).get('description_data',[])
+            text = " ".join(d.get('value','') for d in descs)
+            if not text or text.startswith("** REJECT **"):
+                continue
+            labs = {k:0 for k in FAMILIES}
```

Ruff (0.12.2)

Unnecessary dict comprehension for iterable; use `dict.fromkeys` instead

Replace with `dict.fromkeys(iterable)`)

(C420)